src/main/java/com/github/kokorin/jaffree/ffmpeg/FFmpeg.java

```diff
@@ -363,20 +365,49 @@ public FFmpegResult execute() {
      * @return ffmpeg result future
      */
     public FFmpegResultFuture executeAsync() {
+        return executeAsync(new Executor() {
+            @Override
+            public void execute(Runnable command) {
+                Thread runner = new Thread(command, "FFmpeg-async-runner");
+                runner.setDaemon(true);
+                runner.start();
+            }
+        });
+    }
+    
+    /**
+     * Starts asynchronous ffmpeg execution, executed using the supplied Executor
+     *
+     * @param executor the executor to use for asynchronous execution
+     * @return ffmpeg result future
+     */
+    public FFmpegResultFuture executeAsync(java.util.concurrent.Executor executor) {
```

Comment:
I would suggest to make this method protected. Please, import `java.util.concurrent.Executor`.